lib/Target/AVR/AVRInstrInfo.cpp

```diff
@@ -45,6 +45,30 @@ extern bool RA_InSpillerCode;
 AVRInstrInfo::AVRInstrInfo()
     : AVRGenInstrInfo(AVR::ADJCALLSTACKDOWN, AVR::ADJCALLSTACKUP), RI() {}
 
+// FIXME: BEFORE MERGING: Is there a cleaner tablegen method of doing this?
```

Comment:
To clarify, I was mostly asking because we define these registers like

```
def R25R24 : AVRReg<24, "r25:r24", [R24, R25]>, DwarfRegNum<[24]>;
```

So *something* knows that `R25R24` is composed of `[R24, R25]`. If there was a way to get the "sub registers", then maybe we could avoid some duplication. If we can't, we can't.